### PR TITLE
JS Homework ONLY no HTML and CSS lesson 13, 14, 15

### PR DESCRIPTION
**Pull request by sdmnc**
Hope I got it right

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+let continent = "Europe";
+let population = 2.615;
+
+console.log("Country name is " + country);
+console.log(country + "'s continent is " + continent);
+console.log(country + "'s population is " + population + " billions");
+
+let isIsland = false;
+
+let coutryDescription = country + " is located in " + continent + ", " + population + " mln people live here";
```
Comment (HappyJuliana) [inline script.js]: In “countryDescription” you had to use Template Literals as I think

File: script.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        console.log("D");
+        break;
+    // Anything 59 or below is failing
+    default:
+        console.log("F");
+}
+// Functions //
+
+function TellFortune(n, z, y, x) {
+    console.log("You will be " + x + " in " + y + " , and get married to " + z + " with " + n + " kids")
```
Comment (HappyJuliana) [inline script.js]: Here you also had to use Template Literals

File: script.js [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+if (isInStock) {
+    console.log("The value of good on deposit is " + itemsValue + "$")
+} else {
+    console.log("There are not items in stock")
+}
+
+//  Leap Year//
+
+let year = 2023;
```
Comment (HappyJuliana) [inline script.js]: Here we had to found out the current year by using getFullYear() method
But not to introduce year by ourselves

File: script.js [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+}
+
+// Home -work for lesson #15 //
+
+const NUMBER = 42;
+function isOddEven() {
+    let result = NUMBER / 2
+    // console.log(result);
+    if (result == 0) {
```
Comment (HappyJuliana) [inline script.js]: I would use here strict inequality (===)

**Review by HappyJuliana — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (IvanCeban) [reply script.js]: Technically the solution is correct. 
But agree with Iuliana,  would be more useful to use template literals

File: script.js [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+}
+switch (bobBMI) {
+    case 0:
+        bobBMI < bmiLow;
+        console.log("Bob's BMI is " + bobBMI + " underweighted");
+        break;
+    case 1:
+        bobBMI > bmiLow && bobBMI < bmiMiddle;
+        console.log("Bob's BMI is " + bobBMI + " normal weight");
+        break;
```
Comment (IvanCeban) [inline script.js:50]: A bit different solution from what was required, but good to for using it. Well done

**Review by IvanCeban — APPROVED** [review]
Good job!
Just get the real current year as Iuliana has mentioned

**Review by ElenaF95 — APPROVED** [review]

**Pull request merged**